pages/coaching-dashboard.tsx

```diff
@@ -30,6 +38,40 @@ const coachingDashboard = () => {
       },
     }
   );
+  const enrolledUsers = userList.map((user) => user.id);
```

Comment:
you're not actually using this - you define enrolledUsers again in the useEffect where it's actually used